Makefile

```diff
@@ -239,6 +239,27 @@ ct-mnesia-rocksdb: SYSCONFIG=config/test-lima.config
 ct-mnesia-rocksdb: AETERNITY_TESTCONFIG_DB_BACKEND=rocksdb
 ct-mnesia-rocksdb: internal-ct
 
+ct-stratum: KIND=test
+ct-stratum: SYSCONFIG=config/test.config
+ct-stratum: AETERNITY_TESTCONFIG_DB_BACKEND=mnesia
+ct-stratum: internal-build stratum-client-internal-build
+	@NODE_PROCESSES="$$(ps -fea | grep bin/aeternity | grep -v grep)"; \
+	if [ $$(printf "%b" "$${NODE_PROCESSES}" | wc -l) -gt 0 ] ; then \
+		(printf "%b\n%b\n" "Failed testing: another node is already running" "$${NODE_PROCESSES}" >&2; exit 1);\
+	else \
+		AETERNITY_TESTCONFIG_DB_BACKEND=$(AETERNITY_TESTCONFIG_DB_BACKEND) \
+		$(REBAR) ct --suite apps/aestratum/test/aestratum_integration_SUITE --sys_config $(SYSCONFIG); \
+	fi
+
+stratum-client-internal-build:
+	@(cd ./_build/$(KIND)/ && \
```

Comment:
It looks like you want a specific rebar3 profile (profiles can be combined) for this kind of node?

Reply:
I mean instead of manual git clone.

Reply:
Not sure I follow - do you mean to replace KIND with `test` ?

Reply:
I mean that this fetching of client has room for improvement, and tweaking of rebar.config profiles may be what you want.